site/static/admin/config.yml

```diff
@@ -38,6 +38,23 @@ collections:
       - { name: signedby, label: Signed Off, required: false }
       - { name: tags, label: Tags, widget: "list", required: false,
           hint: "divide tags up using commas, e.g:  emergency, test, contact. The following tags have effects: draft, webchat, popular " }
+      - { label: "Related Departments",
```

Comment:
This creates a single dropdown of departments, to find a related department. Is that what they want or do they want to add all related department**s** which I think there is a widget for as well.